requirements.txt

```diff
@@ -0,0 +1,11 @@
+openai==0.27.8
```

Comment:
We don't need to have this file please delete it (since we have claimed the dependencies in setup.cfg.

Reply:
ok. This file is for hugging face space, and I will add it to .gitignore